libs/ledger-live-common/src/families/stacks/setup.ts

```diff
@@ -0,0 +1,27 @@
+// Goal of this file is to inject all necessary device/signer dependency to coin-modules
+
+import BlockstackApp from "@zondax/ledger-stacks";
+import { Bridge } from "@ledgerhq/types-live";
+import Transport from "@ledgerhq/hw-transport";
+import { createBridges } from "@ledgerhq/coin-stacks";
+import stacksResolver, { signMessage } from "@ledgerhq/coin-stacks/signer/index";
```

Comment:
Shouldn't we keep the `hw-signMessage` ? Because of this https://github.com/LedgerHQ/ledger-live/blob/ccf87549d85730de8e986c0b78bbe20a169ae1ee/libs/ledger-live-common/scripts/sync-families-dispatch.mjs#L112

Reply:
good catch, mapped to fetch the proper file: 

in this commit: 0e7a415b799f6c97bc7fcc1bb49f10dcd65b0145 
<img width="435" alt="Shottr 2024-10-29 19 14 32" src="https://github.com/user-attachments/assets/93e531b5-4fb9-4e66-b066-c50083376631">